app/controllers/api/v2/report_templates_controller.rb

```diff
@@ -162,7 +162,7 @@ def schedule_report
           process_resource_error
         end
       rescue => e
-        render_error 'standard_error', :status => :internal_error, :locals => { :exception => e }
+        render_error 'standard_error', :status => :internal_server_error, :locals => { :exception => e }
```

Comment:
Or should this rather be something like

```ruby
render_error :custom_error, :status => :unprocessable_entity,
                     :locals => { :message => _("Scheduling Report template failed for: %s." % e.message) }
```

Reply:
makes sense, that would be on par with the generate action